core/src/main/java/org/infinispan/globalstate/GlobalConfigurationManager.java

```diff
@@ -49,32 +49,33 @@
     * @param cacheName the name of the configuration
     * @param configuration the configuration object
     * @param flags the flags to apply
+    * @return
     */
-   CompletableFuture<Configuration> createCache(String cacheName, Configuration configuration, EnumSet<AdminFlag> flags);
+   CompletableFuture<Void> createCache(String cacheName, Configuration configuration, EnumSet<AdminFlag> flags);
```

Comment:
Won't this break compatiblity for user code?

Reply:
@ryanemerson I don't see `GlobalConfigurationManager` being exposed to the user, it's just an internal component.